source/sass/blocks/main-nav.scss

```
@@ -0,0 +1,31 @@
+.main-nav {
+  margin: 0;
+  position: absolute;
+  z-index: 5;
+  width: 320px;
+}
+
+.main-nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  background-color: $basic-white;
+}
+
+.menu__item {
+  width: 100%;
+  text-align: center;
+  border-bottom: 1px solid $border-menu;
+}
+
+.menu__link {
+  display: block;
+  padding: 22px 70px 24px 70px;
+  text-decoration: none;
+  color: $basic-black;
+  font-family: "Oswald Medium", "Arial Medium", sans-serif;
```

Общий комментарий по шрифтам будет в fonts
```suggestion
  font-family: "Oswald", "Arial", sans-serif;
```